src/pages/patientView/PatientViewPage.tsx

```diff
@@ -115,22 +112,34 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
         //TODO: this should be done by a module so that it can be reused on other pages
         const qs = queryString.parse((window as any).location.search);
 
+        const reaction1 = reaction(
```

Comment:
The page store in an instance of a library called mobx-react-router.  This handles sychornoization between store and url.  So yes, we only interact with the url via the store.